### PR TITLE
[DO NOT MERGE] Brief recording complete state

### PR DESCRIPTION
**Pull request by microbit-grace**
Feedback needed

### CODE REVIEW

File: src/components/RecordingDialog.tsx [hunk 1]
```diff
@@ -108,8 +110,11 @@ const RecordingDialog = ({
             onDone(data) {
               const recordingId = Date.now();
               addGestureRecordings(gestureId, [{ ID: recordingId, data }]);
-              handleCleanup();
-              onRecordingComplete(recordingId);
+              setRecordingStatus(RecordingStatus.Done);
+              doneTimeout.current = setTimeout(() => {
+                handleCleanup();
+                onRecordingComplete(recordingId);
+              }, 1500);
```
Comment (microbit-matt-hillsdon) [inline src/components/RecordingDialog.tsx:117]: I anticipate that some folks are going to find this quite painful as an extra delay to repeated recording.

Comment (microbit-grace) [reply src/components/RecordingDialog.tsx:117]: Hmmm... yes, not sure what to do about that 🤔 Reduce time, or maybe make it into a full on dialog and add a "do not show again"? Or maybe the new animation for highlighting a newly added recording is sufficient?

Comment (microbit-matt-hillsdon) [reply src/components/RecordingDialog.tsx:117]: I'll experiment with shorter times (+ copy?) when I get a mo.

**Comment by microbit-matt-hillsdon** [comment]
We'll take an adapted version of this change via the recording-options branch. Closing.

**Pull request closed without merging**